src/plugins/eos-external-apps-build-install

```diff
@@ -82,10 +82,11 @@ EXTAPPSBUILDDIR="$HOME/external-apps-build"
 APPDIR="$EXTAPPSBUILDDIR/$APPNAME"
 BUILDDIR="$APPDIR/build"
 BUNDLEDIR="$APPDIR/bundle"
+EXPORTDIR="$BUNDLEDIR/usr"
 
 # Remove any previous left build dir and create it again
 rm -rf "$BUNDLEDIR"
-mkdir -p "$BUILDDIR" "$BUNDLEDIR"
+mkdir -p "$BUILDDIR"
```

Comment:
Not sure this agrees with the commit message... You mention it was being called with the build directory twice, but it was actually called with both the build and bundle directories.